### DISCUSSION TITLE
Add pig/meat farms

### DISCUSSION THREAD

**Discussion by Gbfj1** [Keriew/augustus#452] [Ideas]
The android game is awesome!
Your work is impecable!
But not having pig farms is making me sad, also, in the events held one of them demands 39 meat in stock. Without pig farms its just sitting there for no reason!
Is it possible to add it, please?

**Comment by Areldir** [comment]
Just migrating this over to the Discussions tab.

In terms of the meat requirement for games issue, we will be eventually adding new food types to the game, with associated production chains. This will mean that the requirements for games will be changed to better allow access on all maps. Unfortunately, we can't add meat farms to all maps, as this will adversely impact the balance of housing evolution.

**Comment by Gbfj1** [comment]
Hey Matt!
Thank you for replying so quickly!
But right now i have another pickle in my hands.

Yesterday after playing all day long the game just crashed.
It happened before and my method to solve the issue was uninstall and install it again.

I did a mess and deleted the whole thing from my phone.
I bought the GoG version.
The inno app is not working anymore, is there another app for me to apen the .exe file?

Im so frustrated and pissed at myself... cuz now i cant play.
Can you help me, please?


Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by crudelios** [reply]
@Gbfj1 You can extract it in windows and then copy the files over via USB as well. But inno setup extractor should still work which is strange.

Also, what were you doing when the game crashed? Maybe we can fix the crash.

**Comment by Gbfj1** [comment]
Ok, so,

I have a galaxy note 10+  5g black which is the best on that line.... so you have an scope.

When i play, i normally have the game itself and a pip, netflix, amazon, disney... running

Ive been playing for 4 weeks now and just recently, after playing for a little over 1 hour the phone starts heating up after the 3h mark i have to close everything restart and let it cool. Because the warmth starts bothering my hands and im afraid it will do something bad for the device itself.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
